Model/Dintero.php

```diff
@@ -318,6 +319,7 @@ public function process($merchantOrderId, $transactionId, $sessionId = null)
 
         // if payment has transaction that means that it was already processed
         if ($payment->getLastTransId()) {
+            $this->processTransaction($payment);
```

Comment:
@chaikovskyia @andersem You should reconsider bringing back this line. Getting exception here when Dintero callbacks:
![image](https://user-images.githubusercontent.com/8796678/226308699-70f16b61-a199-45fe-bca1-eaee075befd7.png)
Removing this line makes the callback work again:
![image](https://user-images.githubusercontent.com/8796678/226309288-773ea19e-86c3-4eea-ae42-f285104fffa2.png)

Tested with regular checkout (not express), in both cases transaction is stored in Magento.

Reply:
@mysil2 , thank you. Will check it.